src/molecules/card/index.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+import {
+	Button,
+	Card,
+	CardActionArea,
+	CardActions,
+	CardContent,
+	CardMedia,
+	Collapse,
+	Divider,
+	IconButton,
+	Tooltip,
+	Typography,
+} from "@mui/material";
+import { ExpandMore, LocalOffer, LocationOn, Message, Verified } from "@mui/icons-material";
+import image from "../../../database/images/instructor.png";
+
+// Dummy Data Card
+const name = "Instructor Jesse";
+const location = "Fehmarn";
+const licence = "VDS Level II";
+const price = "50€";
+
+// Dummy Data Card-extended
+const language = ["eng", "ger", "esp"];
+const homespots = ["Gold", "Gollendorf", "Grüner Brink"];
+const slogan = "This is my slogan";
+const introduction =
+	"Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aspernatur blanditiis harum hic illum iste modi optio perspiciatis, quibusdam tempora ut! Architecto incidunt provident quasi quibusdam sapiente sit totam velit voluptatum.";
+const services = ["Pickup", "Material rental"];
+const availabilities = ["Sat", "Sun"];
```

Comment:
Instead of all these `const` you could create an "dummy" object (maybe use an `.json` file) with test data. For example:

```js
const data = [{
  "name": "Instructor Jesse",
  "location": "Fehmarn",
  ...
}]
```

And refer to it via `data[0].name` etc.

Reply:
comming soon